cmd/list.go

```diff
@@ -6,12 +6,16 @@ import (
 	"sort"
 	"text/tabwriter"
 
-	"github.com/whalebrew/whalebrew/packages"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+	"github.com/whalebrew/whalebrew/packages"
 )
 
+var displayHeaders bool
+
 func init() {
+	listCommand.Flags().BoolVarP(&displayHeaders, "display-headers", "", true, "Display column headers for output. Defaults to true.")
```

Comment:
Thanks for the contribution, I would rather go the other way around.
Reason is that it is pretty more intuitive to use `whalebrew list --no-headers` than `whalebrew list --display-headers=false`

```suggestion
	listCommand.Flags().BoolVarP(&hideHeaders, "no-headers", "", false, "Hide column headers for output. Defaults to fase.")
```